vendor/xclannad/wavfile.cc

```diff
@@ -738,7 +742,7 @@ void MP3FILE::Seek(int count) {
 	} while(pimpl->status != MP3FILE_impl::DONE && pimpl->write_pointer < pimpl->write_data_len);
 	return;
 }
-#elif USE_SMPEG
```

Comment:
Uploaded the removal of png, jpeg, and mp3 code.

Thank you for your contribution and for bringing this to my attention.

On Sun, Feb 22, 2015 at 10:13 PM, jbeich <notifications@github.com> wrote:

> In vendor/xclannad/wavfile.cc
> <https://github.com/eglaysher/rlvm/pull/64#discussion_r25145341>:
>
> > @@ -738,7 +742,7 @@ void MP3FILE::Seek(int count) {
> >  	} while(pimpl->status != MP3FILE_impl::DONE && pimpl->write_pointer < pimpl->write_data_len);
> >  	return;
> >  }
> > -#elif USE_SMPEG
>
> Then MAD decoder likely wasn't tested as well. 77540b1
> <https://github.com/eglaysher/rlvm/commit/77540b1a8cb546efd65740bece76edc2099af18e>
> didn't check HAVE_LIBMAD is properly passed towavfile.cc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eglaysher/rlvm/pull/64/files#r25145341>.
>



-- 
-- Elliot